content/changelog/1_15_x_x.mdx

```diff
@@ -0,0 +1,104 @@
+---
+title: 1.15.x.x
+description: Changelog for release v1.15.x.x
+releaseDate: '2022-06-09'
+---
+
+## Added
+
+### Subscribe Webhooks to events
+
+Before this release, all Webhooks were automatically subscribed to all events in a pathway. With this release, you can now subscribe your Webhooks to specific events.
+
+> All legacy webhooks will stay subscribed to all events so there are no breaking changes for customers that already have an integration with Webhooks. However, please note the change in Webhook event type naming in the ["Deprecated"](#deprecated) section below.
+
+### Contextual information in Webhooks and API calls payload
+
+Webhook and API calls now all have contextual information on the pathway & patient in their payload. This information is disclosed through the `pathway` field in the response object.
+
+```json
+{
+  ...,
+  "pathway": {
+    "id": "<PATHWAY_ID>",
+    "pathway_definition_id": "<PATHWAY_DEFINITION_ID>",
+    "patient_id": "<PATIENT_ID>",
+    "start_date": "<START_DATE>"
+  }
+}
+```
+
+### Activity IDs in the reminder created payload
+
+In order to support identifying the activites for which a reminder was created, we now include the activity ids in the corresponding Webhook payload.
+
+```json
+{
+  ...,
+  "reminder": {
+    ...,
+    "new_activities_ids": ["activity_1", "activity_2"]
+  }
+}
+```
+
```

Comment:
https://awellhealth.atlassian.net/browse/AST-3523 won't be in this release so let's remove this.

```suggestion
```